sfcasts/2019_paris/lightning-talks.md

```diff
@@ -5,3 +5,50 @@ Symfony Live Paris 2019 lightning talks by various authors.
 This talk is in French and (sorry!) no transcript is available.
 ***
 
+## Les side-projects, c'est cool ! (@pierstoval)
+
+_Timecode: 00:00_ 
+
+Les _side-projects_, ce sont ces petits projets personnels que l'on fait quand on a du temps libre, en dehors de notre travail.
+
+Parfois, on trouve des idées révolutionnaires, parfois pas.
+
+Il n'y a pas de règles, c'est nous qui les fixons !
+
+Pas de deadline, pas de chef de projet, pas de client, et une liberté totale sur le choix des technos à utiliser !
+
+## Retour critique sur les PSR (@jeremyFreeAgent)
+
+_Timecode: 05:32_ 
```

Comment:
You can turn this into a link with `?playAt=332` (332 being the number of seconds)  to make it so that people can click this and jump straight to that part of the video (it won't auto-play, but when they hit play, it'll be at the right spot)

Reply:
Done :)